deploy.py

```
@@ -40,12 +40,28 @@
 # Optional deploy token (recommended for security).
 # Set via environment: export DEPLOY_TOKEN="your_long_token_from_vps_env"
 DEPLOY_TOKEN: Optional[str] = "6de44dca5425348f2e2ef9456fc820bfe56a5ace68bddeb6da4a1c2a9d9cadc0"
```

Ruff (0.15.15)

[error] Possible hardcoded password assigned to: "DEPLOY_TOKEN"

(S105)